<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1554</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:White;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>650</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Question</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="question_line">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>680</y>
       <width>298</width>
       <height>44</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	background-color: white;
	color: gray;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

}


QLineEdit::hover
{
	background-color:rgb(85, 170, 255);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


QLineEdit::focus
{
	
	color:rgb(0, 0, 0);
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
     </property>
     <property name="text">
      <string>Type a question...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="analyze_button">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>740</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color:rgb(85, 170, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;

}


QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::hover
{
	background-color: rgb(0, 170, 0);

}


QPushButton::pressed
{
	background-color: #3d4ef2;

}</string>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Model Definition</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>371</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Object Detection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="object_detection_combo">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox
{
	background-color: white;
	color: gray;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

}


QComboBox::hover
{
	background-color:rgb(255, 255, 255);
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


QComboBox::focus
{
	
	color: rgb(0, 0, 0);
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Image Captioning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="image_captioning_combo">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox
{
	background-color: white;
	color: gray;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

}


QComboBox::hover
{
	background-color: rgb(85, 170, 255);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


QComboBox::focus
{
	
	color: rgb(0, 0, 0);
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Q&amp;A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="qanda_combo">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox
{
	background-color: white;
	color: gray;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

}


QComboBox::hover
{
	background-color: rgb(85, 170, 255);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


QComboBox::focus
{
	
	color: rgb(0, 0, 0);
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="Tittle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Inference Framework</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>371</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="file_name">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit
{
	background-color: white;
	color: gray;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

}


QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
          </property>
          <property name="text">
           <string>TESTE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="search_file_buttom">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	background-color:rgb(85, 170, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;

}


QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::hover
{
	background-color:rgb(0, 170, 0);

}


QPushButton::pressed
{
	background-color: #3d4ef2;

}</string>
          </property>
          <property name="text">
           <string>Search File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>1121</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 170, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Detected Objects:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="object_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>1071</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 220);
	color: rgb(220, 220, 220);
	font-size: 14px;
	font-weight: bold;
	show-decoration-selected: 0;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>370</y>
       <width>551</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border: 1px solid gray;
	border-radius: 8px;
	background-color: rgb(220, 220, 220);
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>451</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Generated Caption</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="caption_label">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">	border: 1px solid gray;
	border-radius: 8px;
	background-color: rgb(220, 220, 220);
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="question_label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Question:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="answer_label">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">	border: 1px solid gray;
	border-radius: 8px;
	background-color: rgb(220, 220, 220);
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="export_buttom">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>740</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(255, 170, 0);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;

}


QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::hover
{
	background-color:rgb(0, 170, 0);
	border-style: solid;
}


QPushButton::pressed
{
	background-color: #3d4ef2;

}</string>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>300</y>
       <width>447</width>
       <height>64</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Full image</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1554</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
